src/lib/env.js

```diff
@@ -181,13 +174,6 @@ export default () => {
   const maxParallelAdapters = +_env.MAX_PARALLEL_ADAPTERS || 1024;
```

Comment:
Is this still relevant now thats its on lambda? seems like something from the container version

Reply:
maxParallelAdapters is being used in fetch.js and is included in our .env files

Reply:
Hard to say - the impression that i get is that it will default to 2x the cores based on the documentation for the python version. It was set to 1 in production though.